aldryn_apphooks_config/models.py

```diff
@@ -33,9 +34,9 @@ def save(self, *args, **kwargs):
 
     def __str__(self):
         if self.cmsapp:
-            return _('%s / %s') % (self.cmsapp.name, self.namespace)
+            return _('%s / %s') % (escape(self.cmsapp.name), escape(self.namespace))
         else:
-            return _('%s / %s') % (self.type, self.namespace)
+            return _('%s / %s') % (escape(self.type), escape(self.namespace))
```

Comment:
``cmsapp.name`` and ``type`` are not user-input value, and ``__str__`` in general is not meant for printing in the template and anyway it's not marked as safe. Why should we escape it? (I mean Django will escape this by default if used in a template)